### PR TITLE
Docs for manual rolling update

### PR DESCRIPTION
**Pull request by stanlyDoge**
### Type of change
- Documentation

### Description
Docs for manual rolling update

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/book/proc-manual-restart-zookeeper.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+. Annotate `StatefulSet` resource in {ProductPlatformName}.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl annotate`:
+[source,shell,subs=+quotes]
+kubectl annotate statefulset _cluster-name_-zookeeper operator.strimzi.io/manual-rolling-update=true
+endif::Kubernetes[]
++
+On {OpenShiftName} this can be done using `oc annotate`:
```
Comment (laidan6000) [inline documentation/book/proc-manual-restart-zookeeper.adoc]: Same change as above.

File: documentation/book/proc-manual-restart-zookeeper.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+[source,shell,subs=+quotes]
+oc annotate statefulset _cluster-name_-zookeeper operator.strimzi.io/manual-rolling-update=true
++
+. Wait for next reconciliation and for all pods in annotated `StatefulSet` rolling update will be done.
+After all pods are restarted the annotation is removed from `StatefulSet`.
```
Comment (laidan6000) [inline documentation/book/proc-manual-restart-zookeeper.adoc]: Once all the pods have restarted, the annotation is removed from the `StatefulSet`.

File: documentation/book/proc-manual-restart-kafka.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+On {KubernetesName} this can be done using `kubectl annotate`:
+[source,shell,subs=+quotes]
+kubectl annotate statefulset _cluster-name_-kafka operator.strimzi.io/manual-rolling-update=true
+endif::Kubernetes[]
++
+On {OpenShiftName} this can be done using `oc annotate`:
+[source,shell,subs=+quotes]
+oc annotate statefulset _cluster-name_-kafka operator.strimzi.io/manual-rolling-update=true
++
+. Wait for next reconciliation and for all pods in annotated `StatefulSet` rolling update will be done.
```
Comment (laidan6000) [inline documentation/book/proc-manual-restart-kafka.adoc]: _Separate out into a step and an indented paragraph:_

3. Wait for the next reconciliation to occur (every two minutes by default).
     
    All pods are restarted on a rolling basis, as long as the annotation was detected by the reconciliation process.

File: documentation/book/proc-manual-restart-zookeeper.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+On {KubernetesName} this can be done using `kubectl annotate`:
+[source,shell,subs=+quotes]
+kubectl annotate statefulset _cluster-name_-zookeeper operator.strimzi.io/manual-rolling-update=true
+endif::Kubernetes[]
++
+On {OpenShiftName} this can be done using `oc annotate`:
+[source,shell,subs=+quotes]
+oc annotate statefulset _cluster-name_-zookeeper operator.strimzi.io/manual-rolling-update=true
++
+. Wait for next reconciliation and for all pods in annotated `StatefulSet` rolling update will be done.
```
Comment (laidan6000) [inline documentation/book/proc-manual-restart-zookeeper.adoc]: _Change this step as above:_

3. Wait for the next reconciliation to occur (every two minutes by default).

    All pods within the annotated ``StatefulSet`` are restarted on a rolling basis, as long as the annotation was detected by the reconciliation process.

**Comment by laidan6000** [comment]
Added two more comments to clarify the reconciliation step.

File: documentation/book/proc-manual-restart-kafka.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='proc-manual-restart-kafka-{context}']
+= Restarting a Kafka cluster manually
+
+This procedure describes how to manually restart an existing Kafka cluster by using an {ProductPlatformName} annotation.
```
Comment (stanlyDoge) [inline documentation/book/proc-manual-restart-kafka.adoc]: @ppatierno Do we want to say ```manually restart``` or ```manually rolling update```?

Comment (scholzj) [reply documentation/book/proc-manual-restart-kafka.adoc]: `rolling restart`? That is what it really is, right ;-)

Comment (laidan6000) [reply documentation/book/proc-manual-restart-kafka.adoc]: Suggest we change the title to: _Performing a rolling update of a Kafka cluster_

Comment (laidan6000) [reply documentation/book/proc-manual-restart-kafka.adoc]: _Change the first sentence to align with the new terminology:_

This procedure describes how to manually restart a Kafka cluster by using an {ProductPlatformName} annotation. This triggers a rolling update of the pods inside the cluster.

**Pull request merged**